backend/EasySpeak.Core/EasySpeak.Core.BLL/Services/UserService.cs

```diff
@@ -55,7 +55,20 @@ public async Task<UserDto> GetUserAsync()
         {
             var user = await _context.Users.FirstOrDefaultAsync(u => u.Id == _firebaseAuthService.UserId);
 
-            return _mapper.Map<UserDto>(user);
+            var userDto = _mapper.Map<UserDto>(user);
+
+
+            if (user is not null)
```

Comment:
If you want to check for null, first check, then use))

Reply:
You can move getting file from db to a separate private method and return just URL from it